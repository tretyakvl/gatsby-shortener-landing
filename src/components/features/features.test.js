import React from 'react'
import renderer from 'react-test-renderer'
import Features from './features'

describe('Features', () => {
  it('renders correctly', () => {
    const props = {
      title: 'Take a look at our features!',
      desc: 'Our features are very featury',
      list: [
        {
          item: {
            title: 'Cool feature',
            desc: 'This is our favourite'
          }
        },
        {
          item: {
            title: 'Amazing feature',
            desc: 'Well, maybe it`s not true'
          }
        },
        {
          item: {
            title: 'Unique feature',
            desc: 'Call the police if you see someone else list this'
          }
        }
      ]
    }

    const tree = renderer
      .create(<Features {...props} />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
