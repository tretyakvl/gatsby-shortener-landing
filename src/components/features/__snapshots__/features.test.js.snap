// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Features renders correctly 1`] = `
<section
  className="features"
>
  <div
    className="features__container"
    style={
      Object {
        "opacity": 0,
        "transform": "translateX(-20px) translateZ(0)",
      }
    }
  >
    <h2
      className="visually-hidden"
    >
      Shortly features
    </h2>
    <div
      className="features__main"
    >
      <h3>
        Take a look at our features!
      </h3>
      <p>
        Our features are very featury
      </p>
    </div>
    <ul
      className="features__list"
    >
      <li>
        <h3>
          Cool feature
        </h3>
        <p>
          This is our favourite
        </p>
      </li>
      <li>
        <h3>
          Amazing feature
        </h3>
        <p>
          Well, maybe it\`s not true
        </p>
      </li>
      <li>
        <h3>
          Unique feature
        </h3>
        <p>
          Call the police if you see someone else list this
        </p>
      </li>
    </ul>
  </div>
</section>
`;
